bundles/net.tourbook/src/net/tourbook/messages_cs.properties

```diff
@@ -3164,7 +3164,7 @@ tag_view_title_tag                   = Zna\u010Dka:
 tag_view_title_tag_category          = Kategorie:
 
 tourCatalog_view_action_create_left_marker           = Vytvo\u0159it zna\u010Dku v&levo
-tourCatalog_view_action_create_marker                = Vytvo\u0159it &zna\u010Dku
+tourCatalog_view_action_create_marker                = Vytvo\u0159it
```

Comment:
@telemaxx Are you doing now the translations :-)

Reply:
was easy, in this case.
just copy past/from the other messages.properties.

but i keep the shortcuts